Makefile

```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# ScanCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/skeleton for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+# Python version can be specified with `$ PYTHON_EXE=python3.x make conf`
+PYTHON_EXE?=python3
+VENV=venv
+ACTIVATE?=. ${VENV}/bin/activate;
+
+dev:
+	@echo "-> Configure the development envt."
+	./configure --dev
+
+isort:
+	@echo "-> Apply isort changes to ensure proper imports ordering"
+	${VENV}/bin/isort --sl -l 100 src tests setup.py
+
+black:
+	@echo "-> Apply black code formatter"
+	${VENV}/bin/black -l 100 src tests setup.py
+
+doc8:
+	@echo "-> Run doc8 validation"
+	@${ACTIVATE} doc8 --max-line-length 100 --ignore-path docs/_build/ --quiet docs/
+
+valid: isort black
+
+check:
```

Comment:
make isort, make black and make check won't be supported here (?), as we don't have the deps installed (should be in https://github.com/nexB/deltacode/blob/develop/setup.cfg#L44 as the same is available in skeleton). Maybe we should fix also add pycodestyle in the skeleton too (?) while we're at it.

Reply:
Added. Also will open an MR to update in the skeleton.

Reply:
https://github.com/nexB/skeleton/pull/72